docs-src/product/release-stages/public-preview.md

```diff
@@ -0,0 +1,22 @@
+---
+id: public-preview
+title: Public Preview
+description: Learn more about the Public Preview release stage
+sidebar_label: Public Preview
+tags:
+  - product release stages
+  - term
+---
+
+**Access:** Public preview of New Features are available to everyone.
+
+**Classification:** Public preview of New Features may be further developed and tested.
+These New Features are undergoing further refinement and are recommended for production usage.
```

Comment:
Are you missing a "not" regarding production usage? (It may just be my misunderstanding.)

Reply:
No, they are recommended for prod